April23/news.md

```diff
@@ -0,0 +1,9 @@
+# News
+
+## FlashFest organised for April Fools’ snapshot
+
+This month of course comes with April Fools' Day, and that means a Minecraft April Fools' update! In this year's offering we saw a feature in which players regularly vote on "proposals", each of which would make a small change to the game, ranging from changing the colour of grass to... turning the game French? C'est bizarre. Anyway, with this snapshot came an impromptu ModFest, dubbed a "FlashFest". The competition gave contestants 24 hours to make anything they could targeting this snapshot. Some of the brilliant and wildly surreal submissions included a "Bottle of Gender" (what), a mod that allows proposals to be converted to items and then shot out of a gun (???) and a mod with a proposal which instils a sudden generosity into villagers, making them offer all trades for free (the most unrealistic one by far). The competition was a great success, and we hope to see more of these FlashFests in the future - if you want to see all the submissions, go and check out ModFest's Discord (https://discord.gg/gn543Ee).
```

Comment:
What if, Oxford comma :flushed: 

`[...] included a "Bottle of Gender" (what), a mod that allows proposals to be converted to items and then shot out of a gun (???), and a mod with a proposal which instils [...]`

Personally am a fan of Oxford commas (i.e. commas before the and in this case), thoughts?

Reply:
Yeah I agree with you, missed the comma here.
I'll probably write up a style guide soonish